### PR TITLE
Fix ColorChord and add a sinewave monitor.

### PR DESCRIPTION
**Pull request by cnlohr**

### CODE REVIEW

File: main/swadge_esp32.c [hunk 1]
```diff
@@ -323,7 +323,8 @@ void mainSwadgeTask(void* arg __attribute((unused)))
         {
 #if !defined(CONFIG_SWADGE_DEVKIT)
             // If test mode was passed
-            if(getTestModePassed())
+//            if(getTestModePassed())
```
Comment (AEFeinstein) [inline main/swadge_esp32.c:326]: The devkits should pass the test program. Please undo this

File: main/swadge_util.c [hunk 2]
```diff
@@ -165,14 +165,17 @@ paletteColor_t paletteHsvToHex( int16_t hue, int16_t sat, int16_t val)
         val = 0;
     }
 
-    uint32_t rgb = EHSVtoHEXhelper( (uint8_t)hue, sat, val, false );
+    return RGBtoPalette( EHSVtoHEXhelper( (uint8_t)hue, sat, val, false ) );
```
Comment (AEFeinstein) [inline main/swadge_util.c:168]: I thought I had done this already... Maybe in a different branch?

**Review by AEFeinstein — CHANGES_REQUESTED** [review]

File: main/settingsManager.c [hunk 3]
```diff
@@ -261,16 +261,17 @@ bool decMicGain(void)
  */
 uint16_t getMicAmplitude(void)
 {
+    // Using a logarithmic volume control. 
     const uint16_t micVols[] =
     {
         32,
+        45,
```
[the remark this answers is not in the document]
Comment (cnlohr) [reply main/settingsManager.c:268]: No.  Exponential values.  Because audio is logarithmic.

File: main/swadge_esp32.c (hunk 1 above)
Comment (cnlohr) [reply main/swadge_esp32.c:326]: OH CRUD WHOOPS  WILL FIX NOW.

Comment (cnlohr) [reply main/swadge_esp32.c]: fixed.

**Review by AEFeinstein — APPROVED** [review]

**Pull request merged**